internal/util/util.go

```diff
@@ -67,10 +67,10 @@ func ReconcileDesiredBundle(ctx context.Context, c client.Client, bd *rukpakv1al
 
 	// check whether there's an existing Bundle that matches the desired Bundle template
 	// specified in the BI resource, and if not, generate a new Bundle that matches the template.
-	b := CheckExistingBundlesMatchesTemplate(existingBundles, bd.Spec.Template)
+	b := CheckExistingBundlesMatchesTemplate(existingBundles, &bd.Spec.Template)
```

Comment:
Probably best that the downstream consumers don't take a pointer unless `nil` is a valuable input?

Reply:
Updated